components/brave_new_tab_ui/containers/newTab/index.tsx

```diff
@@ -152,6 +183,8 @@ class NewTabPage extends React.Component<Props, State> {
     // if a notification is open at component mounting time, close it
     this.props.actions.showTilesRemovedNotice(false)
     this.imageSource = GetBackgroundImageSrc(this.props)
+    this.sponsoredRichMediaBackgroundInfo = GetSponsoredRichMediaBackground(this.props)
```

Comment:
mega optional: not really sure what to do here but by convention CamelCase functions return a React component while pascalCase are normal functions

However, this file is a bit all over the place, so all good if you want to leave it.

Also, this will not be recalculated as props are updated - not sure if that's okay or not?

Reply:
I would use the same naming as already in the current file, so let's go with GetSponsoredRichMediaBackground, if it works for you.

```
Also, this will not be recalculated as props are updated - not sure if that's okay or not?
```

`this.sponsoredRichMediaBackgroundInfo` is updated in componentDidUpdate(prevProps: Props)